src/main/java/com/google/sps/api/tasks/TasksInterface.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.api.tasks;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.tasks.Tasks;
+import com.google.api.services.tasks.model.Task;
+import com.google.api.services.tasks.model.TaskList;
+import com.google.api.services.tasks.model.TaskLists;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.api.authorization.AuthorizationRequester;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Provides access to the Tasks API.
+ */
+public class TasksInterface {
+  private final Tasks tasksClient;
+
+  public TasksInterface() throws IOException {
+    String userId = UserServiceFactory.getUserService().getCurrentUser().getUserId();
+    Credential credential = AuthorizationRequester.newFlow().loadCredential(userId);
+    tasksClient = new Tasks(
+      AuthorizationRequester.HTTP_TRANSPORT,
+      AuthorizationRequester.JSON_FACTORY,
+      credential);
+  }
+
+  /**
+   * Returns the list of the user task lists.
+   */
+  public List<TaskList> getTasksList() throws IOException {
+    TaskLists tasksLists = tasksClient.tasklists().list().execute();
+    return tasksLists.getItems();
+  }
+
+  /**
+   * Returns the tasks without a date or past their due date
+   * belonging to the task list specified.
+   */
+  public List<Task> getTasks(String tasksListID) throws IOException {
+    List<Task> tasks = tasksClient.tasks().list(tasksListID).execute().getItems();
+    return TasksHelper.filterTasks(tasks);
+  }
+
+  /**
+   * Returns the tasks belonging to the most recently updated task list.
+   */
+  public List<Task> getTasksMostRecentList() throws IOException {
```

Comment:
`getTasksOfMostRecentList`
names are long anyway. So you can as well leave a preposition in there.